frontend/src/components/Footer.tsx

```diff
@@ -0,0 +1,102 @@
+import { MapPin } from "lucide-react";
+
+export default function Footer() {
+  return (
+    <footer className="border-t bg-slate-200 dark:bg-slate-800">
+      <div className="container px-4 py-4 md:py-8 text-slate-800 dark:text-slate-200">
+        <div className="grid gap-8 sm:grid-cols-2 md:grid-cols-4">
+          <div className="space-y-4">
+            <h3 className="text-lg font-semibold">About Nest</h3>
+            <ul className="space-y-2 text-sm">
+              <li>
+                <a className="text-slate-600 hover:text-slate-900 dark:text-slate-400 dark:hover:text-slate-100" href="#">
+                  Our Mission
+                </a>
+              </li>
+              <li>
+                <a className="text-slate-600 hover:text-slate-900 dark:text-slate-400 dark:hover:text-slate-100" href="#">
+                  Team
+                </a>
+              </li>
+              <li>
+                <a className="text-slate-600 hover:text-slate-900 dark:text-slate-400 dark:hover:text-slate-100" href="#">
+                  Careers
+                </a>
+              </li>
+            </ul>
+          </div>
+          <div className="space-y-4">
+            <h3 className="text-lg font-semibold">Resources</h3>
+            <ul className="space-y-2 text-sm">
+              <li>
+                <a className="text-slate-600 hover:text-slate-900 dark:text-slate-400 dark:hover:text-slate-100" href="#">
+                  Contribute
```

Comment:
Could you add corresponding links to the Resources items (e.g. Contribute -> `/projects/contribute`, Projects - `/projects`)?